config/initializers/entry_types.rb

```diff
@@ -1,4 +0,0 @@
-Pageflow.configure do |config|
-  config.plugin(PageflowPaged.plugin)
-  config.plugin(PageflowScrolled.plugin)
-end
```

Comment:
Requiring the host app to register the entry type plugins is sort of a breaking change. Let's postpone that to a point where we continue decoupling the entry type engines from the main gem. For now, we could still let the entry type plugins define the feature flags and make `PageflowPaged` enable its feature flag by default. That way newly generated host apps would still only provide the paged type by default. The changes you did in Dedicated will make the transition smooth for our developers.